pkg/daemon/gateway_linux.go

```diff
@@ -571,6 +571,48 @@ func (c *Controller) setIptables() error {
 			}
 		}
 
+		var ovnSubnetGatewayCountRules []util.IPTableRule
+		_, subnetCidrs, err := c.getDefaultVpcSubnetsCIDR(protocol)
+		if err != nil {
+			klog.Errorf("get subnets failed, %+v", err)
+			return err
+		}
+
+		for name, subnetCidr := range subnetCidrs {
+			ovnSubnetGatewayCountRules = []util.IPTableRule{
+				{Table: "filter", Chain: "FORWARD", Rule: strings.Fields(fmt.Sprintf(`-m comment --comment %s,%s -s %s`, util.OvnSubnetGatewayIptables, name, subnetCidr))},
+				{Table: "filter", Chain: "FORWARD", Rule: strings.Fields(fmt.Sprintf(`-m comment --comment %s,%s -d %s`, util.OvnSubnetGatewayIptables, name, subnetCidr))},
```

Comment:
The `cleanup.sh` script should remove related iptable rules

Reply:
if the subnet is removed, this rule will be removed. its releases in 
![image](https://user-images.githubusercontent.com/47097611/220551579-1289151b-7fbe-4ab5-8f5a-b35bd613f94b.png)